model/cloud.go

```diff
@@ -125,7 +125,6 @@ type Subscription struct {
 	Seats       int      `json:"seats"`
 	Status      string   `json:"status"`
 	DNS         string   `json:"dns"`
-	IsPaidTier  string   `json:"is_paid_tier"`
```

Comment:
We do want to keep passing this based on the discussion today given there are some checks we still need to keep until Nov. 1st. Correct?

Reply:
Correct. I can either leave this open until then, or close the three related PRs and re-open once we reach that date.